transform/gdc/gdcutils.py

```diff
@@ -46,3 +46,12 @@ def query_gdc(endpoint, params):
 
 def extract(data, keys):
     return {key: data.get(key) for key in keys}
+
+
+def get_file(file_id, path):
+    """ download a file from gdc, save in path """
```

Comment:
add a check so that it only downloads the file if a local copy doesn't exist

Reply:
done